packages/components/button/src/Button.tsx

```diff
@@ -1,28 +1,39 @@
 import type { ButtonProps } from './button.types';
-import { ButtonStyled } from './button.styles';
+import {
+  PrimaryButton,
+  SecondaryButton,
+  TertiaryButton,
+  ErrorButton,
+} from './buttonTypes';
 import React from 'react';
-import { useTheme } from '@douro-ui/react';
 
-/**
- * Primary UI component for user interaction
- */
+const buttonTypes = {
+  primary: PrimaryButton,
+  secondary: SecondaryButton,
+  tertiary: TertiaryButton,
+  error: ErrorButton,
+};
+
 const Button = ({
-  primary = false,
-  size = 'small',
-  backgroundColor,
+  typeBtn,
```

Comment:
Type is more accurate and simple.

```suggestion
  type,
```

Reply:
If changing it to "type" causes a TypeScript error, we should consider an alternative name. Do you have any suggestions? Or can we keep "typeBtn"?